### PR TITLE
Adding favicon stuff

### PR DESCRIPTION
**Pull request by Joeoh**

### CODE REVIEW

File: resources/views/index.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?php include('header.php');?>
```
Comment (GavinPacini) [inline resources/views/index.php:1]: Should the header include opening the body? Just so that we don't have a <body> without a </body>?

**Comment by Joeoh** [comment]
Included it to avoid duplication in bodies because it will be shared. I
imagine this having only an index.htnl and an API anyway.

**Comment by GavinPacini** [comment]
@Joeoh, GitHub stripped html from my comments! I meant how the index has a `<body>` but not a `</body>`. It's tiny anyway and you're right, we'll probably just have the index anyway! 😄

**Comment by Joeoh** [comment]
Yeah good point, I'll swap that around or you can if you like.

**Pull request merged**